build.go

```diff
@@ -125,34 +125,7 @@ func main() {
 			}
 
 		case "build-frontend":
-			grunt(gruntBuildArg("build")...)
-
-		case "test":
-			test("./pkg/...")
-			grunt("test")
```

Comment:
This command is referenced in the [documentation](https://github.com/grafana/grafana/blob/main/contribute/developer-guide.md#on-windows) for those who use Windows. I wonder if the removal of the entire command was intentional or it was supposed to be just one line at the bottom. If it was, I will then delete it in the docs.